File: src/app/_modal/model-ticket-type.ts
export class TicketType {
    id : number ;
    ticketType: string;
    description: string;
    workFlow : string ;
    archiveId :string;



  constructor (id :number, ticketType: string,description: string,workFlow : string ,archiveId :string)
  {

    this.id =id;
    this.ticketType=ticketType;
    this.description=description;
    this.workFlow=workFlow;
    this.archiveId=archiveId;
  }
}

File: src/main/java/com/tricon/ticketmanagementsystem/vo/ModelTicketType.java
package com.tricon.ticketmanagementsystem.vo;

public class ModelTicketType {
	
	private int id;
	private String ticketType;
	private String description;
	private String workFlow;
	private String  archiveId;
	
	
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getTicketType() {
		return ticketType;
	}
	public void setTicketType(String ticketType) {
		this.ticketType = ticketType;
	}
	public String getDescription() {
		return description;
	}
	public void setDescription(String description) {
		this.description = description;
	}
	public String getWorkFlow() {
		return workFlow;
	}
	public void setWorkFlow(String workFlow) {
		this.workFlow = workFlow;
	}
	public String getArchiveId() {
		return archiveId;
	}
	public void setArchiveId(String archiveId) {
		this.archiveId = archiveId;
	}
	
	
	

}
